pkg/deploy/api/v1beta3/conversion.go

```diff
@@ -33,35 +28,98 @@ func convert_api_DeploymentConfig_To_v1beta3_DeploymentConfig(in *newer.Deployme
 	if err := s.DefaultConvert(in, out, conversion.IgnoreMissingFields); err != nil {
 		return err
 	}
-	if err := s.Convert(&in.Template, &out.Spec, 0); err != nil {
+	if err := convert_api_DeploymentConfigSpec_To_v1beta3_DeploymentConfigSpec(&in.Spec, &out.Spec, s); err != nil {
 		return err
 	}
-	if err := s.Convert(&in.Triggers, &out.Spec.Triggers, 0); err != nil {
+	if err := convert_api_DeploymentConfigStatus_To_v1beta3_DeploymentConfigStatus(&in.Status, &out.Status, s); err != nil {
 		return err
 	}
-	out.Status.LatestVersion = in.LatestVersion
-	if err := s.Convert(&in.Details, &out.Status.Details, 0); err != nil {
+	return nil
+}
+
+func convert_v1beta3_DeploymentConfigSpec_To_api_DeploymentConfigSpec(in *DeploymentConfigSpec, out *newer.DeploymentConfigSpec, s conversion.Scope) error {
```

Comment:
It's hard for me to keep up with the API conversion stuff in between the times I'm actually working on the API, so at this point I have no reason to believe the manual stuff is necessary. I'd stick with the generator by default unless a problem arises...